routes/review.js

```diff
@@ -0,0 +1,18 @@
+var express=require("express");
+var router=express.Router();
+
+router.get(/reviews?/,(req,res)=>{
+    res.send('get revew')
+});
+router.post(/reviews?/,(req,res)=>{
+    res.send('post revew')
+});
+router.put(/reviews?/,(req,res)=>{
+    res.send('edit revew')
+});
+router.delete(/reviews?/,(req,res)=>{
+    res.send('delete revew')
+});
```

Comment:
Notice that the paths here still point to either `'review/'` or `'reviews/'`. That is ok but problem is, now that we're separating this router from the main app in `app.js`, we'll need to change the paths in this file into just `'/'` otherwise, in `app.js` (which we also need to change in this PR), the routes will have to be accessed as `'/reviews/reviews/'`. 

Also, consider dropping support for including optional `s` since it's not really that critical. Simple route paths would do instead of regex.